app/controllers/application_controller.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
```

Comment:
We'll need to resolve this together.  
GOAL: manage to make AJAX requests with this authentication